src/components/FileList.tsx

```diff
@@ -170,16 +175,23 @@ export class FileList extends React.Component<IFileListProps, IFileListState> {
     event.stopPropagation();
 
     const result = await showDialog({
-      title: 'Discard all changes',
-      body:
-        'Are you sure you want to permanently discard changes to all files? This action cannot be undone.',
-      buttons: [Dialog.cancelButton(), Dialog.warnButton({ label: 'Discard' })]
+      title: this.props.trans.__('Discard all changes'),
+      body: this.props.trans.__(
+        'Are you sure you want to permanently discard changes to all files? This action cannot be undone.'
+      ),
+      buttons: [
+        Dialog.cancelButton(),
```

Comment:
We need to add the translation here too I think

`{ label: this.props.trans.__('Cancel') }` ?